torchmultimodal/models/flava/model.py

```diff
@@ -436,10 +432,9 @@ def flava_model(
     text_hidden_size: int = 768,
     text_num_attention_heads: int = 12,
     text_num_hidden_layers: int = 12,
-    text_hidden_dropout_prob: float = 0.0,
+    text_dropout: float = 0.0,
```

Comment:
these names need not be changed right?

Reply:
it was originally only for the dropout in the MLP in the encoder layer and MHA dropout was a separate argument, but I combined into one. FLAVA does not use either by default.